living/slovak.md

```diff
@@ -1,69 +1,118 @@
 # Web Language: Slovak
 
 Additional names:
+
 - slovenčina
-- 
```

Comment:
Similar to the above comment, please ensure the only changes to the document are URL lines (see the `git diff` for help)